doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-con.adoc

```diff
@@ -0,0 +1,11 @@
+[id='test-scenarios-coverage-report-con']
+= Coverage report for test scenarios
+
+The test scenario designer provides a clear and coherent way of displaying the generated test results using the `Coverage Report` tab from the right side of the test scenario designer.
+
+The `Coverage Report` tab contains the detailed information about available decisions, tested decisions, percentage of tested decisions, how many times each decision has been evaluated, and which decisions are evaluated for the test scenario. You can also view the pie chart with the percentage representation of the ratio between the successful and failed tests.
+
+You can view the coverage reports from the test Scenario designer for both DMN-based scenarios as well as rule-based scenarios.
```

Comment:
I saw you note below about current support only for DMN but I think we should at least move it here. Wdyt?

Reply:
Ack.